addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -785,16 +785,23 @@ var FieldDateRange = InputField.extend({
     },
     /**
      * Bind the scroll event handle when the daterangepicker is open.
+     * Update the begin and end date with the dates from the input values
      *
      * @private
      */
     _onDateRangePickerShow() {
+        const daterangepicker = this.$el.data('daterangepicker');
         this._onScroll = ev => {
             if (!config.device.isMobile && !this.$pickerContainer.get(0).contains(ev.target)) {
-                this.$el.data('daterangepicker').hide();
+                daterangepicker.hide();
             }
         };
         window.addEventListener('scroll', this._onScroll, true);
+        if (this.recordData.planned_date_begin && this.recordData.planned_date_end) {
```

Comment:
this is a framework component... business field names should never been hardcoded here

Reply:
My bad, I've updated the code to be generic.